snforge_std/Scarb.toml

```diff
@@ -10,3 +10,4 @@ re-export-cairo-plugins = ["snforge_scarb_plugin"]
 
 [dependencies]
 snforge_scarb_plugin = { path = "../crates/snforge-scarb-plugin" }
+starknet = "*"
```

Comment:
Just to verify, does that work with minimum supported version of Scarb (2.9.something) that we have?

Reply:
Yes

<img width="740" alt="image" src="https://github.com/user-attachments/assets/dce6420d-d56c-42f3-b6cb-a5bf34a695ae" />